core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/MultithreadedTest.java

```diff
@@ -50,6 +50,7 @@ public static void afterClass() {
 
 	@Test
 	public void testDataAndShapes() {
+		System.out.println("testDataAndShapes");
```

Comment:
The tests in this class are the ones that have been causing the CI to hang. So printing the test method name so it's a bit easier to track back if it happens again.